code/datums/flock/flock.dm

```diff
@@ -578,6 +588,31 @@ var/flock_signal_unleashed = FALSE
 	var/enemy_name = M
 	return (enemy_name in src.enemies)
 
+/datum/flock/proc/addIgnore(atom/A)
+	if (isvehicle(A))
+		for (var/mob/occupant in A)
+			src.addIgnore(occupant)
+	if (ismob(A))
+		var/mob/M = A
+		if (M.find_radio())
+			boutput(A, "<i class='flocksay'>A thousand whispers you didn't realise you were hearing suddenly fall silent as you feel like you are no longer being watched.</i>")
```

Comment:
Not a big issue but feels pretty wordy/length

Reply:
Alternate: A thousand whispers you never realized suddenly fall silent and the feeling that you being watched goes away.